Resources/doc/Sync/StorageManager/storage_manager.rst

```diff
@@ -0,0 +1,28 @@
+==============
+StorageManager
+==============
+
+All commands required for the storage are executing by ``StorageManager``. It has 4 functions that need to be implemented by any type of
```

Comment:
If I understand this correctly,

All commands required for the storage are executed by ``StorageManager``, which implements  ``StorageManagerInterface``. ``StorageManagerInterface`` defines 4 functions that need to be implemented by any type of ``StorageManager``: